routes/capsules.js

```diff
@@ -3,8 +3,29 @@ const db = require("../db-config");
 const capsuleRoutes = require('express').Router();
 
 capsuleRoutes.get("/", (req, res) => {
-  res.send("Welcome to the Capsules page")
-})
+  db.query('SELECT * from capsule', (err, results) => {
+    if (err) {
+      console.log(err);
+      res.status(500);
+    }
+    else {
+      res.status(200).json(results);
+    }
+  })
+});
+
+capsuleRoutes.get("/:id", (req, res) => {
+  console.log(`New request : ${req}`);
+  db.query('SELECT audio_path, audio_title from capsule WHERE id = ?', [req.params.id], (err, results) => {
+    if (err) {
+      console.log(err);
+      res.status(500);
+    }
+    else {
+      res.status(200).json(results);
```

Comment:
good practise is to send back the object, not an array. something like that:

          res.status(200).json(results[0]);